tests-arquillian/pom.xml

```diff
@@ -272,7 +272,7 @@
                                 <arquillian.xml>wildfly-arquillian.xml</arquillian.xml>
                                 <jacoco.agent>${jacoco.agent}</jacoco.agent>
                                 <node.address>${node.address}</node.address>
-                                <additional.vm.args>${additional.vm.args}</additional.vm.args>
+                                <additional.vm.args>${surefire.plugin.jdk9.args}${additional.vm.args}</additional.vm.args>
```

Comment:
BTW, this one is more a paranoia than anything else. We are not yet testing this on WildFly, but I presume that as soon as we will, the same issue will pop up. Therefore, I added it right away.